couchbase/admin/query/v1/query.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto3";
+
+option go_package = "github.com/couchbase/goprotostellar/genproto/admin_query_v1;admin_query_v1";
+option ruby_package = "Couchbase::Protostellar::Generated::Admin::Query::V1";
+option php_namespace = "Couchbase\\Protostellar\\Generated\\Admin\\Query\\V1";
+option java_package = "com.couchbase.client.protostellar.admin.query.v1";
+option java_multiple_files = true;
+
+package couchbase.admin.query.v1;
+
+service QueryAdminService {
+  rpc GetAllIndexes(GetAllIndexesRequest) returns (GetAllIndexesResponse) {}
+
+  rpc CreatePrimaryIndex(CreatePrimaryIndexRequest)
+      returns (CreatePrimaryIndexResponse) {}
+  rpc CreateIndex(CreateIndexRequest) returns (CreateIndexResponse) {}
+
+  rpc DropPrimaryIndex(DropPrimaryIndexRequest)
+      returns (DropPrimaryIndexResponse) {}
+  rpc DropIndex(DropIndexRequest) returns (DropIndexResponse) {}
+
+  rpc BuildDeferredIndexes(BuildDeferredIndexesRequest)
+      returns (BuildDeferredIndexesResponse) {}
+}
```

Comment:
No WatchIndexes?  Is that to be implemented at the SDK level?

Reply:
I originally had 'WatchIndexes' as a method, but then I realized that it's sort of a misnomer, and it's really 'WaitForIndexes' that we expose.  We don't have much existing precedence for having 'Watch' semantics in Protostellar so I omitted it for now in favour of having a discussion on how we should add those mechanisms.

Reply:
Fair enough